tests/testthat/test-outputs.R

```diff
@@ -218,14 +218,45 @@ test_that("calibration options used in summary report if present", {
 
```

Comment:
@jeffeaton Here is the test and an example of how to extract a naomi warning

Reply:
Thanks!